composer.json

```diff
@@ -16,7 +16,7 @@
         "php": "^7.1",
```

Comment:
Collections in version 4 has `[BC] Drop support for PHP 7.1`, so it needs to drop support for 7.1 as well to allow to use them - and I would restrict the new version of api-filter for `^4.0` only, when doing so.